static/src/javascripts/es6/projects/common/modules/crosswords/grid.js

```diff
@@ -1,4 +1,6 @@
 /* jshint newcap: false */
+/* eslint new-cap: 0 */
```

Comment:
react uses capitals, but you don't `new` them, so linting fails. this allows capitals without newing them